doc/api-reference.md

```diff
@@ -204,6 +204,18 @@ Captures the portion of the input that was consumed by the node.
 
 The number of child nodes that the node has.
 
+<b><pre class="api">n.minNumChildren: number</pre></b>
+
+For an iterator node, the minimal number of child nodes possible.
```

Comment:
Well, I spent some time investigating whether minimum or minimal would be right here ;-) but I am happy to change that knowing that a native speaker corrected it.